### DISCUSSION TITLE
Traefik Ingress support

### DISCUSSION THREAD

**Discussion by Sietsedb** [openappsec/openappsec#232] [Feature Requests]
Hey, we're wondering if integration with Traefik Ingress is on the radar? That would be nice!

**Comment by orianelou** [comment]
Hi @Sietsedb, 

Thank you for your interest in open-appsec! While Traefik Ingress is on our roadmap, we currently don't have a specific release date. We appreciate your input and will consider it as we continue to develop open-appsec.

If you would like to contribute to the project by adding support to Traefik Ingress yourself, we will be very glad to support you (or any community member that would like to develop it), yet we encourage you to reach out, using [info@openappsec.io](mailto:info@openappsec.io) before a PR.

**Comment by lordraiden** [comment]
any update about this?

**Comment by gubesch** [comment]
Any news I want to use this on a normal Traefik docker compose installation any possibility?

**Comment by orianelou** [comment]
Hi @gubesch, 

This is on our road map, but we cannot share an expected release date yet.

**Comment by TheLinuxGuy** [comment]
I would really like to see this Traefik middleware module released soon... I'm ditching Nginx Proxy manager for traefik in my homelab.

**Comment by AutomateAaron** [comment]
Would love an idea of where on the roadmap this feature is :)

**Comment by lordraiden** [comment]
No news yet? It has been 2 years in the roadmap

**Comment by nickmaleao** [comment]
Interested also in the traefik integration.

**Comment by llarsson** [comment]
Given how the community is moving away from Ingress NGINX as it gets deprecated, I'm quite certain that a lot of people are looking for a new Ingress controller. And Traefik is turning out to being a popular choice. With that context, it would be nice to know what the status is for the support is, if any, and if there is a plan for a release date.
